src/Utilities/checking_feasibility.jl

```diff
@@ -0,0 +1,22 @@
+function generate_varval(x)
+    return  eval(quote
+        t -> begin
+            return $x[t.value]
+            end
+        end)
+end
```

Comment:
Using `eval` is a sign that something is seriously wrong. You can just go:
```Julia
function generate_varval(x)
    return t -> x[t.value]
end
```

Reply:
There is also a problem that variable `.value` fields need not be contiguous, or ordered from `1:N`.